beacon-node-oapi.yaml

```
@@ -129,7 +131,9 @@ paths:
   /eth/v1/validator/blocks/{slot}:
     $ref: "./apis/validator/block.yaml"
   /eth/v2/validator/blocks/{slot}:
-      $ref: "./apis/validator/block.v2.yaml"
+    $ref: "./apis/validator/block.v2.yaml"
+  /eth/v2/validator/blinded_blocks/{slot}:
```

Isn't this suppose to be :
```suggestion
  /eth/v1/validator/blinded_blocks/{slot}:
```